for-research/public-likelihoods/public-likelihoods.ipynb

```diff
@@ -0,0 +1,936 @@
+      "dPhi_ttbar_Z_Bin_001_mu = 0.4444444444444444  nll= 9477.590868539133\n",
+      "dPhi_ttbar_Z_Bin_001_mu = 0.6666666666666666  nll= 9467.839443849427\n",
+      "dPhi_ttbar_Z_Bin_001_mu = 0.8888888888888888  nll= 9462.106333762014\n",
+      "dPhi_ttbar_Z_Bin_001_mu = 1.1111111111111112  nll= 9460.019964270667\n",
+      "dPhi_ttbar_Z_Bin_001_mu = 1.3333333333333333  nll= 9461.358980950059\n",
+      "dPhi_ttbar_Z_Bin_001_mu = 1.5555555555555554  nll= 9466.01480265484\n",
+      "dPhi_ttbar_Z_Bin_001_mu = 1.7777777777777777  nll= 9473.974311478687\n",
+      "dPhi_ttbar_Z_Bin_001_mu = 2.0  nll= 9485.316708851196\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_3l_WZ) first = 0 last = 5 Likelihood offset now set to 405.768\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_3l_tZq) first = 0 last = 5 Likelihood offset now set to 870.406\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_3l_ttZ) first = 0 last = 5 Likelihood offset now set to 1302.29\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_4l_DF) first = 0 last = 5 Likelihood offset now set to 181.195\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_4l_FF_CR_e_hf) first = 0 last = 1 Likelihood offset now set to 2090.7\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_4l_FF_CR_e_other) first = 0 last = 6 Likelihood offset now set to 1984.6\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_4l_FF_CR_mu_hf) first = 0 last = 1 Likelihood offset now set to 1731.69\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_4l_SF) first = 0 last = 5 Likelihood offset now set to 194.77\n",
+      "[#1] INFO:Minimization -- RooNLLVar::evaluatePartition(reg_4l_ZZ) first = 0 last = 5 Likelihood offset now set to 191.157\n"
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "poi = w.poi()[0]\n",
+    "startVal = poi.getVal()\n",
+    "for val in np.linspace(poi.getMin(),poi.getMax(),10): # scan from min to max allowed value\n",
+    "    poi.setVal(val)\n",
+    "    print(poi.GetName(),\"=\",val,\" nll=\",nll.getVal())\n",
+    "poi.setVal(startVal) # reset value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36018d25-cce4-49c8-b981-7c6e40362a89",
+   "metadata": {},
+   "source": [
+    "# 4. NLL Minimization\n",
+    "\n",
+    "## 4.1. Running an inidividual minimization\n",
+    "\n",
+    "Before running the first minimization with the NLL, the **fit config** should be set as required using both the `nll.fitConfig().MinimizerOptions()` and `nll.fitConfigOptions().SetValue(\"optName\",...)` methods (the latter is used for the 'specific' options). For example, to strange the minimization strategy starting value, you can do: `nll.fitConfig().MinimizerOptions().SetStrategy(...)`. To change the Hesse strategy you can do `nll.fitConfigOptions().SetValue(\"HesseStrategy\",...)`\n"
```

Comment:
Again here, a kinda scary amount of terminology and instruction — if we just need people to trust us, then we should just say "You need to run this"; if we want to try to explain what they're doing (change the strategy to...), then we need a little more explanation I think.